ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -19,6 +21,14 @@ use crate::{
 use std::{collections::HashMap, ffi::CString, sync::Arc};
 use ykaddr::addr::symbol_to_ptr;
 
+/// The mode for processing blocks.
+enum ProcessMode {
+    /// Process blocks normally.
+    Normal,
+    /// Don't process any remaining blocks (except in order to count promotions/debugstrs etc).
+    Skip,
```

Comment:
Is this to do with outlining? We already have a mechanism for skipping blocks during outlining, and I wonder if we are duplicating that here?